FreeRTOS/Test/CMock/makefile.in

```diff
@@ -40,12 +40,15 @@ INCLUDE_DIR         +=  -I$(C_EXCEPTION_SRC_DIR)
 CPPFLAGS    :=
 
 # Compiler flags
-CFLAGS              :=  $(INCLUDE_DIR)  -O0 -ggdb -pthread  --std=c99 -Werror -Wall
+CFLAGS              :=  $(INCLUDE_DIR)  -pthread  --std=c99 -Werror -Wall
 CFLAGS              +=  -fstack-protector-all
 CFLAGS              +=  -Wformat -Werror=format-security -Werror=array-bounds
-CFLAGS              +=  -D_FORTIFY_SOURCE=2
+CFLAGS              +=  -D_FORTIFY_SOURCE=0
 ifeq ($(ENABLE_SANITIZER),1)
     CFLAGS          +=  -fsanitize=address,undefined -fsanitize-recover=address
+    CFLAGS              +=  -O3 -D_FORTIFY_SOURCE=2
+else
+    CFLAGS              :=  $(INCLUDE_DIR)  -O0 -ggdb 
```

Comment:
This will override the settings on line 43.

Reply:
oh i will fix that
this is since my gcc  was complaining that fortify source and O0 are mutually exclusive